src/common/backup.thrift

```diff
@@ -154,3 +154,15 @@ struct configuration_query_restore_response
     2:list<dsn.error_code>  restore_status;
     3:list<i32>             restore_progress;
 }
+
+struct start_backup_apps_request
+{
+    1:string        backup_provider_type;
+    2:set<i32>      app_ids;
```

Comment:
I prefer each RPC handling only one app. This can make the implementation simpler.